mysql-test/suite/innodb/t/percona_sys_tablespace_encrypt.test

```diff
@@ -221,9 +222,30 @@ ALTER TABLE t9 TABLESPACE=`innodb_system`;
 --let $grep_output= boolean
 --source include/grep_pattern.inc
 
+--replace_result $MYSQLD_DATADIR1 MYSQLD_DATADIR1
+--let $restart_parameters="restart: --datadir=$MYSQLD_DATADIR1 --innodb-sys-tablespace-encrypt=ON"
+--source include/start_mysqld.inc
+
+--source include/count_sessions.inc
```

Comment:
This and wait_until_count_sessions are redundant due to kill_mysqld between them

Reply:
Ok, fixed